### PR TITLE
OSDOCS-4323: Using CPU Manager and Using Topology Manager updates

### PR DESCRIPTION
**Pull request by ahardin-rh**
Version(s):
4.9+

Issues:
https://issues.redhat.com/browse/OSDOCS-4323
https://issues.redhat.com/browse/OSDOCS-4324

Link to docs preview:

https://55893--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/using-cpu-manager.html

### CODE REVIEW

File: scalability_and_performance/using-cpu-manager.adoc [hunk 1]
```diff
@@ -16,4 +16,18 @@ CPU Manager is useful for workloads that have some of these attributes:
 * Coordinate with other processes and benefit from sharing a single processor
 cache.
 
+Topology Manager collects hints from the CPU Manager, Device Manager, and other Hint Providers to align pod resources, such as CPU, SR-IOV VFs, and other device resources, for all Quality of Service (QoS) classes on the same non-uniform memory access (NUMA) node.
+
+Topology Manager uses topology information from collected hints to decide if a pod can be accepted or rejected on a node, based on the configured Topology Manager policy and pod resources requested.
```
Comment (mburke5678) [inline scalability_and_performance/using-cpu-manager.adoc:21]: s/from collected/from the collected

**Comment by mburke5678** [comment]
@ahardin-rh One tiny, ignorable nit. Otherwise LGTM

File: scalability_and_performance/using-cpu-manager.adoc (hunk 1 above)
Comment (ahardin-rh) [reply scalability_and_performance/using-cpu-manager.adoc]: Fixed. Thanks!

**Review by rsevilla87 — APPROVED** [review]
LGTM!

**Pull request merged**